protocol/blockchain/transaction/transaction-signing.md

```diff
@@ -0,0 +1,67 @@
+# Transaction Signing
+
+Generally, every input of a  must contain one or more signatures so that the transaction is valid. This applies to any input whose previous output [locking script](/protocol/blockchain/transaction/locking-script) includes one of the following [operation codes](/protocol/blockchain/script#operation-codes-opcodes): `OP_CHECKSIG`, `OP_CHECKSIGVERIFY`, `OP_CHECKMULTISIG`, `OP_CHECKMULTISIGVERIFY`.
+
+Generally, every input of a [transaction](/protocol/blockchain/transaction) require one or more signature. The signatures enforce (sign) what the transaction looks like and make it impossible for a third party to temper with without invalidating the transaction. 
```

Comment:
Did you append the line again instead of replacing? 😅

Reply:
Oops!